src/codeu/chat/server/contexts/UserContext.java

```diff
@@ -0,0 +1,67 @@
+package codeu.chat.server.contexts;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.BasicView;
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.User;
+import codeu.chat.contexts.ConversationContext;
+import codeu.chat.security.ConversationSecurityDescriptor;
+import codeu.chat.security.ConversationSecurityFlags;
+import codeu.chat.util.Uuid;
+
+public class UserContext extends codeu.chat.contexts.UserContext {
+
+  public UserContext(User user, BasicView view, BasicController controller) {
+    super(user, view, controller);
+  }
+  
+  @Override
+  public ConversationContext start(String name) {
+    return super.start(name);
+  }
+
+  @Override
+  public Iterable<ConversationContext> conversations() {
+    final Collection<ConversationContext> all = new ArrayList<>();
+    for (final ConversationHeader conversation : view.getConversations()) {
+      if (conversation.security.hasFlags(user.id, ConversationSecurityFlags.VIEW_MESSAGES)) {
+        all.add(new ConversationContext(user, conversation, view, controller));
+      }
+    }
+    return all;
+  }
+
+  @Override
+  public String getAllConvosFromServer(Uuid signedInId) {
```

Comment:
I believe these are overridden for the "future" addition of security in the interests part.